porch/Makefile

```diff
@@ -93,7 +94,8 @@ start-function-runner:
 	  --network=porch \
 	  --ip 192.168.8.202 \
 	  --name function-runner \
-	  $(IMAGE_REPO)/$(PORCH_FUNCTION_RUNNER_IMAGE):$(IMAGE_TAG)
+	  $(IMAGE_REPO)/$(PORCH_FUNCTION_RUNNER_IMAGE):$(IMAGE_TAG) \
+	  -disable-runtimes pod
```

Comment:
> [nit] --disable-runtimes (lead with 2 hyphens?)

The function runner is built with golang native flag package (https://pkg.go.dev/flag) which use single hyphen.
The command built with the `cobra` package uses 2 hyphens.
IIRC most of our porch command is built with golang native flag package (https://pkg.go.dev/flag).

>More conceptual - what if this had the opposite polarity?

What you suggested works. I picked `disable-runtimes` since I prefer to use empty string to be the default.
I don't have a strong opinion about flag naming. I can change it to `--runtimes` if it make more sense.

Reply:
wow, good catch re: cobra vs. golang. no need to change. thanks for educating me!